packages/reports/tests/dummy/config/environment.js

```diff
@@ -33,7 +33,11 @@ module.exports = function(environment) {
     navi: {
       user: 'navi_user',
       defaultTimeGrain: 'day',
-      dataSources: [{ name: 'dummy', uri: 'https://data.naviapp.io', type: 'bard-facts' }],
+      defaultDataTable: 'network',
```

Comment:
i'm not sure what you mean?

Reply:
Something like this
```js
{
  defaultTimeGrain: 'day',
  defaultDataTable: 'network',
  dataSources: [
    { name: 'dummy', uri: 'https://data.naviapp.io', type: 'bard-facts' },
    { 
      name: 'blockhead', 
      uri: 'https://data2.naviapp.io', 
      type: 'bard-facts', 
      defaultDataTable: 'myTable' 
    }
  ],
}
```
since dataTable and timeGrain could easily be invalid across different data sources

Reply:
Maybe have a default request object for when we make a new report/widget... that's what this is **really** for.